csharp/ql/src/semmle/code/csharp/dataflow/Nullness.qll

```diff
@@ -176,7 +176,17 @@ private predicate isMaybeNullArgument(Ssa::ExplicitDefinition def, MaybeNullExpr
     pdef = def.getADefinition()
   |
     p = pdef.getParameter().getSourceDeclaration() and
-    p.getAnAssignedArgument() = arg and
+    arg = p.getAnAssignedArgument() and
+    not arg.getEnclosingCallable().getEnclosingCallable*() instanceof TestMethod
+  )
+}
+
+private predicate isNullDefaultArgument(Ssa::ExplicitDefinition def, MaybeNullExpr arg) {
```

Comment:
Either change `MaybeNullExpr` to `AlwaysNullExpr`, or change `because the parameter has a null default value` to `because the parameter has a potentially null default value` further down.